problemsets-mat223/modules/module4.tex

```diff
@@ -138,7 +138,15 @@
 	Let $\vec a=\mat{1\\2}$, $\vec b=\mat{3\\3}$, $\vec c=\mat{2\\1}$, and $\vec v=\mat{3\\4}$. Which vector out of
 	$\vec a$, $\vec b$, and $\vec c$ has a direction closest to the direction of $\vec v$?
 
-	XXX Finish
+	To find the vector, we need to compute the angles $\alpha$, $\beta$ and $\gamma$ between vectors $\vec a$, $\vec b$, $\vec c$ and $\vec v$ respectively. 
+	
+	Similarly to the example above, equating the algebraic and geometric definitions of the dot product, we see
```

Comment:
-> By equating the geometric and algebraic definitions of the dot product, we know \[\cos\theta = ...\] where $\theta$ is the anlbe between vectors $\vec p$ and $\vec q$. Therefore,